### PR TITLE
[MetricsAdvisor] Deprecation of package

### PR DESCRIPTION
**Pull request by l0lawrence**
setup.py classifier is Inactive and sdk_packaging.toml has auto_update=false already

### CODE REVIEW

File: sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,9 @@
 # Release History
 
-## 1.1.0 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.0.1 (2024-08-27)
```
Comment (l0lawrence) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md:3]:
```suggestion
## 1.0.1 (Unreleased)
```

**Review by xiangyan99 — COMMENTED** [review]
I don't see the setup.py change to mark it as Inactive?

**Comment by l0lawrence** [comment]
> I don't see the setup.py change to mark it as Inactive?

It is already marked as inactive -- I was just about to post in reviews :)

File: sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,9 @@
...
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.0.1 (Unreleased)
 
 ### Other Changes
+- This package is no longer being maintained, as the service has been retired. There is no replacement package for this package.
 - This version and all future versions will require Python 3.7+. Python 2.7 and 3.6 are no longer supported.
```
Comment (xiangyan99) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md:7]: We only need 
"This package is no longer being maintained, as the service has been retired. There is no replacement package for this package."

Let's delete the other two.

**Review by xiangyan99 — APPROVED** [review]

File: sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md [hunk 3]
```diff
@@ -1,16 +1,9 @@
 # Release History
 
-## 1.1.0 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.0.1 (Unreleased)
```
Comment (swathipil) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md:3]: release date?

Comment (swathipil) [reply sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md:3]: oh unless you're not planning on not releasing today

File: sdk/metricsadvisor/azure-ai-metricsadvisor/README.md [hunk 4]
```diff
@@ -15,15 +15,15 @@ Metrics Advisor is a scalable real-time time series monitoring, alerting, and ro
 
 ## _Disclaimer_
```
Comment (swathipil) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/README.md:16]: you should remove everything except the disclaimer and this line at the top:
`# Microsoft Azure SDK for Python`

Comment (l0lawrence) [reply sdk/metricsadvisor/azure-ai-metricsadvisor/README.md:16]: ah I didn't realize it was remove everything my bad, I will fix :)

Comment (l0lawrence) [reply sdk/metricsadvisor/azure-ai-metricsadvisor/README.md:16]: and I see where it said that in the doc too oof

File: sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md (hunk 3 above)
Comment (l0lawrence) [reply sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md:3]: was going to leave it and release sometime next week :)

File: sdk/metricsadvisor/azure-ai-metricsadvisor/README.md (hunk 4 above)
Comment (swathipil) [reply sdk/metricsadvisor/azure-ai-metricsadvisor/README.md]: no worries, I just updated it actually to make it clearer!

File: sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/_version.py [hunk 5]
```diff
@@ -4,6 +4,6 @@
 # Licensed under the MIT License.
 # ------------------------------------
 
-VERSION = "1.1.0"
+VERSION = "1.0.1"
```
Comment (iscai-msft) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/_version.py:7]: why is the version going down to `1.0.1`

File: sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md (hunk 3 above)
Comment (l0lawrence) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md:3]:
```suggestion
## 1.0.1 (2024-08-26)
```

File: sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/_version.py (hunk 5 above)
Comment (l0lawrence) [reply sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/_version.py:7]: You only bump to a patch release for a deprecation @swathipil

Comment (swathipil) [reply sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/_version.py:7]: The last released version was 1.0.0 @iscai-msft. As per previous guidelines as well, we do a patch bump for deprecation.

**Review by swathipil — APPROVED** [review]
lgtm! Can you mention in the description of the PR that the setup.py classifier is Inactive and sdk_packaging.toml has auto_update=false already?

Will be easier to track in the future.

**Pull request merged**